frontend/src/features/admin-form/responses/ResponsesPage/ResponsesPageSkeleton.tsx

```diff
@@ -0,0 +1,15 @@
+import { Box, Container, Skeleton, Stack } from '@chakra-ui/react'
+
+export const ResponsesPageSkeleton = (): JSX.Element => (
```

Comment:
any reason we used a fresh component instead of using `Skeleton` inside `SecretKeyVerification`?

Reply:
This skeleton is needed for when we are still loading the form in the first place. Thus we don't have any information on whether it is an email form or storage form yet. It wouldn't make sense to display the secret key verification input skeleton, so we will display a fully general skeleton instead.

(Actually, there are already `Skeleton`s in `SecretKeyVerification`, which are conditioned on `isLoading` from `StorageResponsesContext`. But they won't end up being used since `isLoading` in `StorageResponsesContext` is directly taken from `useAdminForm`. We can probably get rid of those.)

Reply:
if there already is a skeleton, for `SecretKeyVerification`, why do we need a skeleton for this page?

Reply:
cos `SecretKeyVerification` is rendered when form is loaded and determined to be in storage mode. But until the form is loaded, we don't know whether to show the storage mode or email mode version of the responses page. So as I mentioned, we should show a generic skeleton rather than having the secret key verification skeleton

Reply:
gotcha